com.nolimitconnect.NoLimitConnect.yml

```diff
@@ -0,0 +1,41 @@
+
+app-id: com.nolimitconnect.NoLimitConnect
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: nolimitconnect
+finish-args: 
+    - --device=all
```

Comment:
why device all?

Reply:
It is multimedia ie camera, microphone recording/playback etc. There does not seem to be a specification for these.